src/html/runnabear.html

```diff
@@ -1,47 +1,151 @@
 <!-- section -->
 <section id="bears" class="section">
 
-  <h1 class="h1">Runnabear</h1>
-
-  <h2 class="h2">Full Image</h2>
-
+  <h1 class="h1 weight-strong">Runnabear</h1>
+  <h2 class="h2 weight-strong">Full Image</h2>
   <div class="figure-wrapper">
```

Comment:
The figure wrapper needs to use flexbox and wrap so that all the figures can be the same height.